framework/source/class/qx/ui/basic/Image.js

```diff
@@ -108,6 +108,24 @@ qx.Class.define("qx.ui.basic.Image",
       apply : "_applyScale"
     },
 
+    /**
+     * Whether to preserve the image ratio (ie prevent distortion), and which dimension
+     * to prioritise
+     */
+    forceRatio : {
+      init : 'auto',
```

Comment:
There was some semblance of scaling already. Does 'auto' here match the prior behavior? If not, we should start right now keeping track of what's not BC for our release notes.

Reply:
the previous version of "scaling" was just stretching the image; I've had this code in use for a while, I think that the "auto" setting was one that I chose because IMHO a stretched image is almost never useful, so the stretching was not useful.  You are right that this is a BC issue, I've added an entry in the changelog